tree/tree/src/TTree.cxx

```diff
@@ -9578,6 +9615,8 @@ void TTree::UseCurrentStyle()
 Int_t TTree::Write(const char *name, Int_t option, Int_t bufsize) const
 {
    FlushBasketsImpl();
+   if (R__unlikely(option & kOnlyPrepStep))
+      return 0;
```

Comment:
Isn't zero an error code for `TObject::Write` in general? This might break code checking for the return value of `TTree::Write()`.

Reply:
The code will (eventually) be migrated from `Write` to `Merge` avoid this odd return value pattern.